Libraries/Lists/VirtualizedList.js

```diff
@@ -1581,7 +1637,7 @@ class VirtualizedList extends React.PureComponent<Props, State> {
     if (
       width > 0 &&
       height > 0 &&
-      this.props.initialScrollIndex > 0 &&
+      this.props.initialScrollIndex &&
```

Comment:
This change is unrelated, but fixed a flow error. Maybe related to onStartReached PR?